stable/rabbitmq-ha/templates/NOTES.txt

```diff
@@ -3,8 +3,13 @@
   Credentials:
 
     echo Username      : {{ .Values.rabbitmqUsername }}
+    {{ if .Values.secret.name }}
```

Comment:
I might be looking at something old but where does this come from, I only the following in the values.yaml:
```
secret:
  autogenerate: true
```